client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -98,6 +98,10 @@ const ListTileSubtitle = styled.div`
 	align-items: center;
 `;
 
+const NoIcon = styled.div`
+	font-size: xx-large;
+`;
```

Comment:
Let's prefer CSS objects instead of string soups:

```suggestion
const NoIcon = styled.div({
	fontSize: 'xx-large'
})
```

CSS objects give us typing and syntax highlighting out of the box. It's much more TS friendly too 👍